lib/vsc/administration/user.py

```diff
@@ -136,6 +138,18 @@ def __init__(self, user_id, storage=None, pickle_storage='VSC_SCRATCH_KYUKON', r
         super(VscTier2AccountpageUser, self).__init__(user_id, rest_client, account=account,
                                                       pubkeys=pubkeys, use_user_cache=use_user_cache)
 
+        # Move to vsc-config?
+        default_pickle_storage = {
+            GENT: VSC_SCRATCH_KYUKON,
+            BRUSSEL: VSC_SCRATCH_THEIA,
+        }
+
+        if host_institute is None:
+            host_institute = GENT
+
+        if pickle_storage is None:
+            pickle_storage = default_pickle_storage[host_institute]
+
         self.pickle_storage = pickle_storage
         if not storage:
             self.storage = VscStorage()
```

Comment:
i would add 2 extra attrs (and even get rid of `self.storage`)
```
self.institute_path_templates = self.storage.path_templates[self.host_institute]
self.institute_storage = self.storage[self.host_institute]
```
and use those below